packages/lisk-transactions/src/transactions/helpers/get_id.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import * as cryptography from '@liskhq/lisk-cryptography';
+import { TransactionJSON } from '../../transaction_types';
+import { getTransactionBytes } from '../../utils';
+
+export const getId = (transaction: TransactionJSON): string => {
```

Comment:
we have the same function in `utils/get_transaction_id.ts`

Reply:
see below answer